igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildCache.groovy

```diff
@@ -38,16 +38,18 @@ class JenkinsCache {
     @Value('${spinnaker.jedis.prefix:igor}')
     String prefix
 
+    String id = 'builds'
+
     List<String> getJobNames(String master) {
         Jedis resource = jedisPool.resource
-        List<String> jobs = resource.keys("${prefix}:${master}:*").collect { extractJobName(it) }.sort()
+        List<String> jobs = resource.keys("${prefix}:${id}:${master}:*").collect { extractJobName(it) }.sort()
```

Comment:
@tomaslin I solved the prefix thing in another way for the buildcache https://github.com/gardleopard/igor/commit/7f0746e690f090ad286966050752c83600595e7f#diff-5ab6267632f5cde16014ea8f70e6ee56R59

This makes all travis hosts appear as travis-${name} everywhere.